tests/ui/associated-types/hr-associated-type-bound-param-6.stderr

```diff
@@ -17,6 +17,22 @@ LL |     <(i32,) as X<i32>>::f("abc");
    |
    = help: the trait `X<'_, T>` is implemented for `(S,)`
 
+error[E0277]: the trait bound `for<'b> i32: X<'b, i32>` is not satisfied
```

Comment:
We have a duplicated error here b/c we can only refine one of two copies of this fulfillment error: one that comes from well-formedness, and one that comes from the predicates of.

I think it's fine to ignore this regression, since it's a net improvement for other errors.